site/cypress/e2e/Cypress-Feature/grade_inquiries.spec.js

```diff
@@ -50,6 +62,45 @@ describe('Test cases revolving around grade inquiries', () => {
             cy.get('[data-testid="reply-text-area-36"]').type('Submitty');
             cy.get('[data-testid="markdown-mode-tab-preview"]').first().should('exist');
             cy.get('[data-testid="grade-inquiry-actions"]').contains('Submit Grade Inquiry').should('not.be.disabled');
+
+            // need to clear local storage to refresh grader's responsibility page
+            cy.clearLocalStorage();
+        });
+
+        // student view
+        cy.login('beahaf');
+        cy.visit(['sample', 'gradeable', gradeableId]);
+        cy.get('[data-testid="grade-inquiry-container"]').should('contain.text', 'Grade inquiries are due by 01/01/9998 @ 12:00 AM EST');
+    });
+
+    it('should test cases regarding abnormal grade inquiry dates', () => {
+        const beforeGradeInquiryStartDate = '1970-01-01 00:00:00';
+        cy.login();
+        setGradeInquiriesForGradeable(gradeableId, beforeGradeInquiryStartDate);
+        cy.get('[data-testid="grade-inquiry-dates-warning"]').should('be.visible');
+        cy.get('[data-testid="grade-inquiry-dates-warning"]').should('have.text', 'Warning: Grade Inquiry ends before it starts. Students will not be able to make Grade Inquires.');
+
+        // TA / Instructor view
+        cy.visit(['sample', 'gradeable', gradeableId, 'grading', 'details']);
+        cy.get('[data-testid="view-sections"]').click();
+        cy.get('[data-testid="grade-button"]').eq(2).click();
+        cy.get('[data-testid="grade-inquiry-info-btn"]').click();
+        cy.get('[data-testid="invalid-grade-inquiry"]').should('exist');
+        cy.get('[data-testid="invalid-grade-inquiry"]').should('have.text', 'Grade inquiries will not start. Contact the instructor if this is unexpected.');
+        cy.get('[data-testid="component-tab-36"]').should('not.exist');
+
+        // student view
+        cy.login('beahaf');
+        cy.visit(['sample', 'gradeable', gradeableId]);
+        cy.get('[data-testid="grade-inquiry-container"]').should(($el) => {
+            expect($el.text().trim()).to.equal('There will be no grade inquiries. Contact the instructor if this is a mistake.');
         });
```

Comment:
Is this different than 
```suggestion
        cy.get('[data-testid="grade-inquiry-container"]').should('contain', 'There will be no grade inquiries. Contact the instructor if this is a mistake.');
        });
```

Reply:
when i tried to trim it, there was a couple new lines after. I originally wanted it to exactly equal beacuse I accidentally combined two parts of a twig {% if %} but could not find a way to do it